### PR TITLE
Update Badges / README.md

### PR DESCRIPTION
**Pull request by athix**
Moved from / closes [#797](https://github.com/NoamB/sorcery/pull/797)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+[![Gem Version](https://badge.fury.io/rb/sorcery.svg)](https://rubygems.org/gems/sorcery)
+[![Gem Downloads](https://img.shields.io/gem/dt/sorcery.svg)](https://rubygems.org/gems/sorcery)
 [![Build Status](https://travis-ci.org/NoamB/sorcery.svg?branch=master)](https://travis-ci.org/NoamB/sorcery)
 [![Code Climate](https://codeclimate.com/github/NoamB/sorcery.png)](https://codeclimate.com/github/NoamB/sorcery)
 [![Inline docs](http://inch-ci.org/github/NoamB/sorcery.png?branch=master)](http://inch-ci.org/github/NoamB/sorcery)
-
-# sorcery
+[![Join the chat at https://gitter.im/NoamB/sorcery](https://badges.gitter.im/join_chat.svg)](https://gitter.im/NoamB/sorcery?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ## MAINTAINER NEEDED
```
Comment (athix) [inline README.md]: The maintainer needed section can probably be removed now that @Ch4s3 has agreed to take over. Should that be a part of this PR?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
+# Sorcery: Magical Authentication
+
+[![Gem Version](https://badge.fury.io/rb/sorcery.svg)](https://rubygems.org/gems/sorcery)
+[![Gem Downloads](https://img.shields.io/gem/dt/sorcery.svg)](https://rubygems.org/gems/sorcery)
 [![Build Status](https://travis-ci.org/NoamB/sorcery.svg?branch=master)](https://travis-ci.org/NoamB/sorcery)
```
Comment (athix) [inline README.md]: Change to `Sorcery/sorcery`? Also applies to lines 6 and 7. (Possibly more elsewhere too.)

**Comment by athix** [comment]
I think I managed to setup all the open source services to work with the new repo. Should be good to merge now, @Ch4s3, thoughts?

**Comment by Ch4s3** [comment]
Sorry for the delay.

**Pull request merged**